template_feed/Microsoft.DotNet.Common.ItemTemplates/content/GlobalJson/.template.config/template.json

```diff
@@ -100,6 +100,26 @@
           "displayName": "Disable roll-forward"
         }
       ]
+    },
+    "TestRunner": {
+      "type": "parameter",
+      "description": "The test runner to use.",
+      "displayName": "Test runner",
+      "replaces": "TEST_RUNNER_VALUE",
+      "defaultValue": "",
+      "datatype": "choice",
+      "choices": [
+        {
+          "choice": "VSTest",
+          "description": "Use the VSTest test runner",
```

Comment:
Maybe something like `Use VSTest as test runner` or `Use VSTest test runner` might be easier to read.

Reply:
Fixed